cmd/jujud/agent/upgrade_test.go

```diff
@@ -560,6 +561,9 @@ func (s *UpgradeSuite) TestUpgradeSkippedIfNoUpgradeRequired(c *gc.C) {
 }
 
 func (s *UpgradeSuite) TestDowngradeOnMasterWhenOtherStateServerDoesntStartUpgrade(c *gc.C) {
+	if runtime.GOOS == "windows" {
+		c.Skip("issue 1403084: doesn't work on windows because of symlink issue")
```

Comment:
this needs to be checked out. we should be able to downgrade. But with a glacial priority....we need to have a state machine on windows first